checks/evaluation/security_policy.go

```diff
@@ -15,10 +15,59 @@
 package evaluation
 
 import (
+	"fmt"
+
 	"github.com/ossf/scorecard/v4/checker"
 	sce "github.com/ossf/scorecard/v4/errors"
 )
 
+func scoreSecurityCriteria(contentLen, linkedContentLen, urls, emails, discvuls int) (int, string) {
+	score := 0
+	reason := ""
+
+	// #1: found one linked (email/http) content: score += 3
+	//     rationale: someone to collaborate with or link to
+	//     information (strong for community)
+	if urls >= 1 || emails >= 1 {
+		score += 3
+		reason += "linked content, "
```

Comment:
Instead of building the reason, you can use `log.Info()` for things that are positive, and `log.Warn()` for things that are negative. Something along the lines of:
```golang
if urls >= 1 || emails >= 1 {
  log.Info("Found link in description: %s", link)
} else {
  log.Warn("no email or URL found in description)
}
```

Also note that the logging function allows you to pass file / line numbers as well, which you should try to use. These are reflected in the SARIF format (GH scanning dashboard). They are also accessible via `-show-details` and in the weekly scan.

Same comment for the other checks below in the code.

Reply:
this point along with the related `SecurityPolicyFileInformation` noted in one of the other suggested changes sounds perfect (wish I'd thought of it). I'll make that change.

Reply:
log.Info/Warn - done